client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -124,6 +126,7 @@ class PacmanGame extends Component {
 
   setGameStatus = (status) => {
     if (status === 'finish') {
+      leaveGame();
       clearInterval(this.animationHandler);
       this.setState({ status: 2 });
```

Comment:
Set status values as constants, by this we can know which number correspond to which status

Reply:
Planning to add more statues in my next PR, to pause, resume and restart the game.